configs/repos/ufs/packages/ufs-weather-model-debug-env/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from spack import *
+
+class UfsWeatherModelDebugEnv(BundlePackage):
+    """Development environment for ufs-weathermodel-bundle"""
+
+    homepage = "https://github.com/ufs-community/ufs-weather-model"
+    git      = "https://github.com/ufs-community/ufs-weather-model.git"
+
+    maintainers = ['kgerheiser', 'climbfuji']
+
+    version('main', branch='main')
+
```

Comment:
@kgerheiser I leave it up to you whether you want to repeat what is in `ufs-weather-model-env` except the esmf-debug and mapl-debug lines or not. If the rest is repeated, a user can install only the `ufs-weather-model-debug-env` or the `ufs-weather-model-env`. If only esmf+debug and mapl+debug are listed here, then the user needs to install both to get the debug version. I prefer the latter, because most users just want everything to run the ufs-weather-model, in both debug and release mode.

Reply:
I like it like this